test/extensions/quic_listeners/quiche/BUILD

```diff
@@ -182,18 +162,13 @@ envoy_cc_test(
 envoy_cc_test(
     name = "active_quic_listener_test",
     srcs = ["active_quic_listener_test.cc"],
-    # Skipping as quiche quic_stream_send_buffer.cc does not currently compile on Windows
-    tags = [
-        "nofips",
-        "skip_on_windows",
-    ],
+    tags = ["nofips"],
     deps = [
         ":quic_test_utils_for_envoy_lib",
         ":test_utils_lib",
         "//source/extensions/quic_listeners/quiche:active_quic_listener_config_lib",
         "//source/extensions/quic_listeners/quiche:active_quic_listener_lib",
         "//source/extensions/quic_listeners/quiche:envoy_quic_utils_lib",
-        "//source/extensions/quic_listeners/quiche:udp_gso_batch_writer_lib",
```

Comment:
@danzh2010 should I try reverting the block above in source/e/q/q/BUILD and also guard this in a select? Or is there a better component to target now in this deps list?

Reply:
Can you try reverting the changes in active_quic_listener_lib and also remove this line here?